pkg/Cpanel/Security/Advisor/Assessors/Brute.pm

```diff
@@ -54,13 +55,64 @@ sub _check_for_brute_force_protection {
 
     }
     elsif ( -e "/etc/csf" ) {
-        $security_advisor_obj->add_advice(
-            {
-                'type' => $Cpanel::Security::Advisor::ADVISE_GOOD,
-                'text' => ['CSF is installed'],
+        if ( -e "/etc/csf/csf.disable" ) {
+            if ( -e "/usr/local/cpanel/whostmgr/docroot/cgi/configserver/csf" ) {
+                $security_advisor_obj->add_advice(
+                    {
+                        'type'       => $Cpanel::Security::Advisor::ADVISE_BAD,
+                        'text'       => ['CSF is installed, but appears to be disabled.'],
+                        'suggestion' => [
+                            'Click “Firewall Enable“ in the “[output,url,_1,ConfigServer Security & Firewall,_2,_3]” area. Alternately, run “csf -e“ from the command line.',
+                            $self->base_path('cgi/configserver/csf.cgi'),
+                            'target',
+                            '_blank'
+                        ],
+                    }
+                );
             }
-        );
-
+            else {
+                $security_advisor_obj->add_advice(
+                    {
+                        'type'       => $Cpanel::Security::Advisor::ADVISE_BAD,
+                        'text'       => ['CSF is installed, but appears to be disabled.'],
+                        'suggestion' => ['Run “csf -e“ from the command line.'],
+                    }
+                );
+            }
+        }
+        elsif ( check_lfd_running() ) {
+            $security_advisor_obj->add_advice(
+                {
+                    'type' => $Cpanel::Security::Advisor::ADVISE_GOOD,
+                    'text' => ['CSF is installed, and LFD is running.'],
+                }
+            );
+        }
+        else {
+            if ( -e "/usr/local/cpanel/whostmgr/docroot/cgi/configserver/csf" ) {
```

Comment:
There's probably a more cPanel correct way to determine if the csf plugin for WHM is present. This is one of the directories that the install script creates, and I couldn't imagine any reason why anything else would create a directory with this name, so it seems safe to use as a test. That being said, if there are recommendations for a better method to test that this plugin is present, I can't imagine that it would hurt to use them instead.